### ISSUE TITLE
Setting primitive values via transform can disrupt other values on the instance

### ISSUE THREAD

**Issue by jcarpenter** [aframevr/aframe#262]
In the sphere primitive I'm setting the `segments-height` to be half of `segments`. 

I then specify segments on a sphere instance:

``` html
<!-- Orange -->
<a-sphere position="0 4 -5" radius="4" color="#F16745" segments="5">
    <a-image position="0 -4 0" src="../_images/radial-shadow-3.png" rotation="-90 0 0" scale="4 4 4"></a-image>
</a-sphere>

```
The result is as follows:

![screenshot](https://cloud.githubusercontent.com/assets/1425497/11611160/d6f4ad22-9b73-11e5-8010-24acf939481b.png)

For the orange sphere instance, the rendered radius reports "4" in DOM inspector, but visible radius is more like 0.2. If I re-enter "4" in the DOM inspector (or any value) the radius starts working again and matches the value.

If I manually specify `segments-height` on the instance in the source html, the problem does not occur.

So it seems that `setAttribute` via transform is somehow colliding with attributes already manually set by the user.

The code for the transform is as follows:

``` js
  mappings: {
    value: {
      radius: 'geometry.radius',
      segments: 'geometry.segmentsWidth',
      'segments-height': 'geometry.segmentsHeight',
      color: 'material.color',
      opacity: 'material.opacity',
      shader: 'material.shader',
      transparent: 'material.transparent',
      metalness: 'material.metalness',
      roughness: 'material.roughness',
      src: 'material.src'
    }
  },

  transforms: {
    value: {
      segments: function (value) {
        this.setAttribute('segments-height', value / 2);
        return value;
      }
    }
  }

```

**Comment by cvan** [comment]
@ngokevin: can you confirm this is fixed in PR #883?

**Comment by ngokevin** [comment]
This is fixed in the PR, but I'm closing this issue since the new primitives never landed anywhere.
